tools/mo/openvino/tools/mo/graph/port.py

```diff
@@ -274,6 +274,29 @@ def get_tensor_names(self, port_renumber: bool = False):
                     tensor_names_list.append(tensor_name.replace(',', '\\,'))
         return sorted(tensor_names_list)
```

Comment:
nit: can we remove this sort so that a developer can set the order

Reply:
No. This sort is needed because serializing/deserializing from IR does not preserve order. Probably this should be fixed as a separate bug. So if we remove this sort IR reader will read names with different order each run and IR comparator tests won't pass.